test-browser/commands/elementIsNotPresent.js

```diff
@@ -0,0 +1,15 @@
+const EventEmitter = require('events')
+
+class ElementIsPresent extends EventEmitter {
```

Comment:
Nightwatch api methods contains commands that check for element present and element not present [Nightwatch API](https://nightwatchjs.org/js/app/api/method.html), so there may be no need to implement custom commands for those assertions. Except you have other reasons for wanting to use a custom command.